packages/conf-llvm/conf-llvm.3.9/files/source-linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main'
+curl -L http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Well, it's the current design of ```opam-depext```. I can't change that :/
The ```"source"```` predicate is applied when no native package is available. In this case it is mainly used for the CI to work.